interface/src/app/signature/high_range_exp.tpl.html

```diff
@@ -2,12 +2,10 @@
   <h4>
     <b>Experiments with the Highest Range: </b>
   </h4>
-  <button class="btn btn-primary" ng-click="setMode()"
-          ng-hide="topMode">Show Top {{topNum}}</button>
-  <button class="btn btn-primary" ng-click="setMode()"
-          ng-show="topMode">Show All</button>
-
-  <br>
+  <p>
+    Explore gene expression experiments with the highest range of this
+    signature's activity.
+  </p>
   (Click the activity distribution widget to see sample annotations in this experiement.)
```

Comment:
I'm not sure this needs to be a parenthetical statement. It could just be the second sentence in the `<p>` above.